components/ModEditHealthcareProfessionalSection.vue

```diff
@@ -0,0 +1,279 @@
+                    :key="`${specialty}-${index}`"
+                    :value="specialty"
+                >
+                    {{ specialty }}
+                </option>
+            </select>
+            <label
+                for="Locales"
+                class="my-2 text-primary-text text-sm font-bold font-sans"
+            >
+                {{ $t("modHealthcareProfessionalSection.selectLocales") }}
+            </label>
+            <select
+                id="healthcare-professional-locales"
+                v-model="sectionFields.healthcareProfessionalLocales"
+                data-testid="mod-healthcare-professional-section-spoken-locales"
+                name="Locales"
+                multiple
+                class="mb-5 px-3 py-3.5 w-96 h-32 bg-secondary-bg rounded-lg border border-primary-text-muted
+                            text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+            >
+                <option
+                    v-for="(locale, index) in localeOptions"
+                    :key="`${locale}-${index}`"
+                    :value="locale"
+                >
+                    {{ locale }}
+                </option>
+            </select>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { onMounted } from 'vue'
+import { ModerationScreen, useModerationScreenStore } from '~/stores/moderationScreenStore'
+import { useHealthcareProfessionalsStore } from '~/stores/healthcareProfessionalsStore'
+import { Locale, type LocalizedNameInput, Insurance, Degree, Specialty } from '~/typedefs/gqlTypes'
+import { multiSelectWithoutKeyboard } from '~/utils/multiSelectWithoutKeyboard'
+import SVGTrashCan from '~/assets/icons/trash-can.svg'
+import SVGProfileIcon from '~/assets/icons/profile-icon.svg'
+
+const moderationScreenStore = useModerationScreenStore()
+const healthcareProfessionalsStore = useHealthcareProfessionalsStore()
+// less clear name necessary because of filter rules on line length (max = 130)
+const professionalNameArray = healthcareProfessionalsStore.healthcareProfessionalSectionFields.healthcareProfessionalNameArray
+const sectionFields = healthcareProfessionalsStore.healthcareProfessionalSectionFields
+
+// Creating the necessary parts for the multi-select
+const insuranceOptions = Object.values(Insurance) as Insurance[]
+const extractInsuranceOptions = (option: HTMLOptionElement): Insurance => option.value as Insurance
+const degreeOptions = Object.values(Degree) as Degree[]
+const extractDegreeOptions = (option: HTMLOptionElement): Degree => option.value as Degree
+const specialtyOptions = Object.values(Specialty) as Specialty[]
+const extractSpecialtyOptions = (option: HTMLOptionElement): Specialty => option.value as Specialty
+const localeOptions = Object.values(Locale) as Locale[]
+const extractLocaleOptions = (option: HTMLOptionElement): Locale => option.value as Locale
```

Comment:
I like the idea of what you have going on here. It was a little confusing understanding it at first, because I think the naming syntax `extractInsuranceOptions` etc doesn't really describe what the variables are doing. For example, we aren't returning the Insurance _options_, we're returning the `Insurance` that is stored in `option.value`. Also, it's just a single `Insurance` so the pluralized `Options` give a false impression of there being multiple items.